lib/shopify_app/sessions_concern.rb

```diff
@@ -87,5 +90,16 @@ def return_address
       session.delete(:return_to) || main_app.root_url
     end
 
+    def perform_after_install_actions
+      return unless ShopifyApp.configuration.enable_after_install_actions
```

Comment:
instead of setting this configuration to `true` or `false` can we default it to `false` and set it to a job class when it should be enabled?

then this would look like:

```ruby
def perform_after_install_actions
  return unless ShopifyApp.configuration.after_install_job
  ShopifyApp.configuration.after_install_job.perform_later(shop_domain: session[:shopify_domain])
end
```

I like that you added a generator for the Job though we should keep that even though this connection point will be truly generic.